content/svenska/dashboards/wastewater/covid_quantification/covid_quant_GU.md

```diff
@@ -55,9 +53,13 @@ Wang, H., Churqui, M.P., Tunovic, T., Enache, L., Johansson, A., Karmander, A.,
 
 ## Metoder
 
-Insamling av avloppsvatten sker genom en fast insamlare som samlar in 30ml avlopps vatten per 10,000m3  av inkommande avloppsvatten. För analys veckovis poolas sju prover (varje avloppsvatten prov representerar insamling under ett dygn). Veckoprovet består av 1.5-15l avloppsvatten (beroende av flödet) som skickas till Klinisk Mikrobiologi vid Sahlgrenska Universitetssjukhuset för analys. Analys sker på måndagen i veckan efter provinsamling.
+Insamling av avloppsvatten sker genom en fast insamlare som samlar in 30ml avlopps vatten per 10,000m3 av inkommande avloppsvatten. För analys veckovis poolas sju prover (varje avloppsvatten prov representerar insamling under ett dygn). Veckoprovet består av 1.5-15l avloppsvatten (beroende av flödet) som skickas till Klinisk Mikrobiologi vid Sahlgrenska Universitetssjukhuset för analys. Analys sker på måndagen i veckan efter provinsamling.
```

Comment:
avlopps vatten -> avloppsvatten 
 avloppsvatten prov ->  avloppsvattenprov

Reply:
changed in new version